cmd/scrambled-eggs/scramble.py

```diff
@@ -74,7 +74,18 @@ def should_skip_key(key):
         "$minKey",
         "$oid",
         "$regularExpression",
-        "$timestamp"
+        "$timestamp",
```

Comment:
Only string values get scrambled, so you should be able to remove order, count_over_threshold, alias_queue, and length from this list. I double checked by running it on the same file to verify that this was the case